apps/dash-svm/app.py

```diff
@@ -425,13 +425,27 @@ def update_svm_graph(
     return [
         html.Div(
             id="svm-graph-container",
-            children=[dcc.Graph(id="graph-sklearn-svm", figure=prediction_figure)],
+            children=dcc.Loading(
+                className="graph-wrapper",
+                children=dcc.Graph(id="graph-sklearn-svm", figure=prediction_figure),
+                style={"display": "none"},
+            ),
         ),
         html.Div(
             id="graphs-container",
             children=[
-                dcc.Graph(id="graph-line-roc-curve", figure=roc_figure),
-                dcc.Graph(id="graph-pie-confusion-matrix", figure=confusion_figure),
+                dcc.Loading(
+                    className="graph-wrapper",
+                    children=dcc.Graph(id="graph-line-roc-curve", figure=roc_figure),
+                    style={"display": "none"},
```

Comment:
Why is this here? I think it will just hide the loading indicator, which is not what we want (we want for the app to show when it's loading something).